os/kernel/binary_manager/binary_manager.c

```diff
@@ -161,15 +161,15 @@ int binary_manager(int argc, char *argv[])
 	/* Create binary manager message queue */
 	g_binmgr_mq_fd = mq_open(BINMGR_REQUEST_MQ, O_RDWR | O_CREAT, 0666, &attr);
 	if (g_binmgr_mq_fd == (mqd_t)ERROR) {
-		bmdbg("Failed to open message queue\n");
+		bmdbg("Fail to open message queue, errno %d. EXIT!\n", errno);
```

Comment:
Let's use same word, MQ instead of message queue.
Please refer `bmdbg("Fail to receive mq, ret %d, errno %d, retry!\n", nbytes, errno);`